notebooks/MAST/PanSTARRS/PS1_DR2_TAP/PS1_DR2_TAP.ipynb

```diff
@@ -0,0 +1,898 @@
+    "\n",
+    "Note that the results are restricted to objects with `nDetections>1`, where `nDetections` is the total number of times the object was detected on the single-epoch images in any filter at any time.  Objects with `nDetections=1` tend to be  artifacts, so this is a quick way to eliminate most spurious objects from the catalog.\n",
+    "\n",
+    "This query runs in TAP's asynchronous mode, which is a queued batch mode with some overhead and longer timeouts, useful for big catalogs like PanSTARRS. It may not be necessary for all queries to PS1 DR2, but the TAP+ client can automatically handle the additional processing required over synchronous mode."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "job = TAP_service.launch_job_async(\"\"\"\n",
+    "SELECT objID, RAMean, DecMean, nDetections, ng, nr, ni, nz, ny, gMeanPSFMag, rMeanPSFMag, iMeanPSFMag, zMeanPSFMag, yMeanPSFMag\n",
+    "FROM dbo.MeanObjectView\n",
+    "WHERE\n",
+    "CONTAINS(POINT('ICRS', RAMean, DecMean),CIRCLE('ICRS',187.706,12.391,.2))=1\n",
+    "AND nDetections > 1\n",
+    "  \"\"\")\n",
+    "TAP_results = job.get_results()\n",
+    "TAP_results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Get DR2 light curve for RR Lyrae star KQ UMa\n",
+    "\n",
+    "This time we start with the object name, use the MAST name resolver (which relies on Simbad and NED) to convert the name to RA and Dec, and then query the PS1 DR2 mean object catalog at that position.  Then we run a spatial query to TAP using those coordinates."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "objname = 'KQ UMa'\n",
+    "ra, dec = resolve(objname)\n",
+    "radius = 1.0/3600.0 # radius = 1 arcsec\n",
+    "\n",
+    "query = \"\"\"\n",
+    "SELECT objID, RAMean, DecMean, nDetections, ng, nr, ni, nz, ny, gMeanPSFMag, rMeanPSFMag, iMeanPSFMag, zMeanPSFMag, yMeanPSFMag\n",
+    "FROM dbo.MeanObjectView\n",
+    "WHERE\n",
+    "CONTAINS(POINT('ICRS', RAMean, DecMean),CIRCLE('ICRS',{},{},{}))=1\n",
+    "AND nDetections > 1\n",
+    "\"\"\".format(ra,dec,radius)\n",
+    "print(query)\n",
+    "\n",
+    "job = TAP_service.launch_job_async(query)\n",
+    "TAP_results = job.get_results()\n",
+    "TAP_results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get Detection Information\n",
+    "\n",
+    "TAP_service = TapPlus(url=\"http://vao.stsci.edu/PS1DR2/tapservice.aspx\")Extract all the objects with the same object ID from the [Detection](https://outerspace.stsci.edu/x/b4Oc) table, which contains all the individual measurements for this source. The results are joined to the [Filter](https://outerspace.stsci.edu/x/nIOc) table to convert the filter numbers to names."
```

Comment:
Not sure if this reference to the code was meant to be here. Either remove if not needed or add spacing so “Extract” isn’t rendered as a URL.